nixos/modules/services/desktop-managers/plasma6.nix

```diff
@@ -249,10 +249,11 @@ in {
 
     xdg.portal.enable = true;
     xdg.portal.extraPortals = [
+      kdePackages.kwallet
       kdePackages.xdg-desktop-portal-kde
       pkgs.xdg-desktop-portal-gtk
```

Comment:
```suggestion
      (pkgs.xdg-desktop-portal-gtk.override { buildPortalsInGnome = false; })
```
I've done this locally to remove some gnome dependencies. I think all the functionality still works as before.

Reply:
This no longer exists. See github.com/NixOS/nixpkgs/pull/345979

Reply:
Even if this option wasn't removed, setting it would make the gtk portal useless as buildPortalsInGnome was disabling the settings portal.

Reply:
Yes someone on Matrix was hitting issues with not having this portal, and specifically the settings portal, just a week or so ago, which is why this was added for Plasma in the first place. Upstream recommends including it.

Reply:
@mjm it was me :)

Reply:
Wasn't not having to depend on components of a DE you don't use the whole point of xdg-desktop-portal existing in the first place?

Reply:
No, it exists because flatpak people are afraid of same connection use in DE D-Bus APIs